content/en/docs/installation/compatibility.md

```diff
@@ -40,9 +40,13 @@ To address this, the webhook can be run in the host network so it can be reached
 by cert-manager, by setting the `webhook.hostNetwork` key to true on your
 deployment, or, if using Helm, configuring it in your `values.yaml` file.
 
-Note that since kubelet uses port `10250` by default on the host network, the
-`webhook.securePort` value must be changed to a different, free port.
+**IMPORTANT**: kubelet on AWS uses port `10250` by default on the host network, which
+clashes with the default port for the cert-manager webhook.
 
+As such, the `webhook.securePort` value must be changed to a different, free port when
```

Comment:
Suggestion: specify that `webhook.securePort` is a Helm chart parameter and suggest how to do the same with standard manifests (seems to be passing a flag and setting a port by looking at [the Helm template](https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/templates/webhook-deployment.yaml))

I guess it might be nice to do the same with `webhook.hostNetwork` above.